src/components/endpoints/Filesystem.js

```diff
@@ -45,14 +47,22 @@ export const updateInfoType = (path, infoType) => {
 
 /**
  * Get a paged directory listing
+ * @param key - Query key for react-query
  * @param path - The resource path
  * @param rowsPerPage - The number of rows per page
  * @param orderBy - A column to order items by
  * @param order - Ascending or Descending
  * @param page - The page number or offset
  * @returns {Promise<any>}
  */
-export const getPagedListing = (path, rowsPerPage, orderBy, order, page) => {
+export const getPagedListing = (
+    key,
+    path,
+    rowsPerPage,
+    orderBy,
+    order,
+    page
+) => {
     return callApi({
         endpoint: `/api/filesystem/paged-directory?path=${encodeURIComponent(
             path
```

Comment:
I moved all the interactions with service / callApi to `serviceFacade` folders mainly for 2 reason

- Keep the `components` folder exclusively for react components.
- Avoid digging thru myriad of components folders to find this `endpoints` folder.

I am don't mind the folder being called `endpoints` or `serviceFacade`.

Reply:
I didn't realize you had a separate serviceFacade folder - I'll move my stuff there